playbook/app/pb_kits/playbook/pb_badge/_badge.tsx

```diff
@@ -67,8 +69,10 @@ const Badge = (props: BadgeProps): React.ReactElement => {
               {...closeProps}
           >
             <Icon
+                className="svg-inline--fa"
+                customIcon={timesIcon.icon as unknown as { [key: string]: SVGElement }}
                 fixedWidth
-                icon="times"
+                icon=""
```

Comment:
I'm guessing there's some reasoning we had to leave icon="" here, and on all the other kits as well. Any particular reason we can't just remove all these? @markdoeswork

Reply:
It breaks the icon kit not to provide the icon prop. I didn't really look into why

Reply:
done